package.json

```diff
@@ -23,14 +23,14 @@
   },
   "license": "Apache-2.0",
   "scripts": {
-    "clean": "rm -rf dist/ types/",
-    "build": "npm run clean && rollup --config rollup.config.mjs && npm run build:types",
+    "clean": "node -e \"['dist','types'].forEach(d=>require('fs').rmSync(d,{recursive:true,force:true}))\"",
+    "build": "rollup --config rollup.config.mjs && npm run build:types",
```

Comment:
doesn't removing clean here mean build can have long living artifacts... I don't think this should have been dropped and moved to prepublishOnly